Plugin/Magento/GiftCard/GenerateGiftCardAccountsOrderPlugin.php

```diff
@@ -31,7 +31,7 @@ public function aroundExecute(
         /** @var Order $order */
         $order =  $event->getOrder();
         if ($order->getPayment()->getMethod() === BoltPayment::METHOD_CODE &&
-            in_array($order->getStatus(), [Order::STATE_PENDING_PAYMENT, Order::STATE_CANCELED])) {
+            in_aray($order->getStatus(), [Order::STATE_PENDING_PAYMENT, Order::STATE_CANCELED])) {
```

Comment:
Did you change this accidentally?

Reply:
No, this was just to force an error to test the notifications. I've cleaned that up now